### DISCUSSION TITLE
Option to create desktop shortcut for a specific floorp profile

### DISCUSSION THREAD

**Discussion by androidghosh** [Floorp-Projects/Floorp#1461] [Feature Request]
Floorp can create different browser profiles from the handy profile option in the toolbar. But each time I need to open floorp with the other profile I had to jump through a lot of hoops. 
What I want is the ability to create a desktop shortcut to the other profile directly, within the handy profile option in the toolbar. So that when I open the shortcut from desktop, it opens floorp with the other profile.

**Comment by notDavid** [comment]
Fyi, not sure what OS you are running on, but this is probably already possible:

https://wiki.mozilla.org/Firefox/CommandLineOptions#-no-remote
